### ISSUE TITLE
Function to return gated sample

### ISSUE THREAD

**Issue by andreas-wilm** [whitews/FlowKit#63]
**Is your feature request related to a problem? Please describe.**
In my case I need to apply all WSP defined gates to an input FCS and export the gated samples to FCS files. 

**Describe the solution you'd like**
It would be nice if there was a helper function that would make this easy. Alternatively, it would be good to cover this in the tutorial.

**Describe alternatives you've considered**
I think, the only alternative is to do this manually, which is a bit clunky and error prone (and probably incomplete). For example:

```
import flowkit as fk

fks_fj = fk.Session(...)
fks_fj.import_flowjo_workspace(...)

sample_group = fks_fj.get_sample_groups()[0]# or as appropriate
fks_fj.analyze_samples(sample_group)
sample_id = ...
gr = fks_fj.get_gating_results(sample_group, sample_id)
sample = fks_fj.get_sample(sample_id)
df = sample.as_dataframe(source="raw")
gate = fks_fj.get_sample_gates(sample_group, sample_id)[0]# or as appropriate
df_gated = df[gr.get_gate_indices(gate.id)]
df_gated.columns = df_gated.columns.droplevel()# see https://github.com/whitews/FlowKit/issues/62
gated_sample = fk.Sample(df_gated)
gated_sample.export(..., source='raw')
```

**Comment by whitews** [comment]
Hi Andreas,

I've added the Session method `get_gate_events` that returns a Pandas DataFrame containing only the gated events. It also takes optional arguments for a comp matrix and a transform to easily get processed event data. I decided to return a DataFrame instead of a new Sample instance because it is likely that users will want the gated event data to perform additional statistical analyses. Plus, it is trivial to create a new Sample from the DataFrame with the fix implemented recently.

Hopefully this will make it less cumbersome to retrieve gated events.

Regards,
Scott
